sirbot.yml

```diff
@@ -2,13 +2,17 @@
 - name: deploy sir-bot-a-lot
   hosts: sirbot
   vars:
+    repo: https://github.com/pyslackers/sirbot-pythondev.git
+    version: master
     username: "{{ sirbot_user }}"
     domain: "{{ sirbot_domain }}"
     service_name: sirbot
     site_port: "{{ sirbot_port }}"
     exec_start: "/home/{{ username }}/.pyvenv/bin/sirbot -c {{ sirbot_config }}"
     work_dir: "/home/{{ username }}/"
     www: false
+    install_self: true
+    migration: "/home/{{ username }}/.pyvenv/bin/sirbot -c {{ sirbot_config }} --update"
```

Comment:
Everything looks good, though one clarifying question - is this really just a "post install" script to run? I hadn't even though about it before, but doing this in a handler is an excellent idea. My only feedback is maybe we call it "post_install_command" or just "post_install" since for the website it would probably be the migrations and for you it's the `sirbot update` command.